src/rdma.c

```diff
@@ -1537,7 +1539,7 @@ int connRdmaListen(connListener *listener) {
         bindaddr = default_bindaddr;
     }
 
-    listener->priv = rdma_listener = zcalloc_num(bindaddr_count, sizeof(*rdma_listener));
+    rdma_listeners = rdma_listener = zcalloc_num(bindaddr_count, sizeof(*rdma_listener));
```

Comment:
So priv is not used anymore?

Reply:
It will be used for `serverRdmaContextConfig` in the next step (for RDMA builtin).